diff --git a/services/surfaceflinger/Android.mk b/services/surfaceflinger/Android.mk
index ec296d3..4a9bfa6 100644
--- a/services/surfaceflinger/Android.mk
+++ b/services/surfaceflinger/Android.mk
@@ -1,4 +1,7 @@
 LOCAL_PATH:= $(call my-dir)
+
+ifneq ($(BOARD_HAVE_OWN_SURFACEFLINGER),true)
+
 include $(CLEAR_VARS)
 
 LOCAL_SRC_FILES:= \
@@ -57,6 +60,8 @@ LOCAL_MODULE:= libsurfaceflinger
 
 include $(BUILD_SHARED_LIBRARY)
 
+endif
+
 ###############################################################
 # uses jni which may not be available in PDK
 ifneq ($(wildcard libnativehelper/include),)
